### PR TITLE
Get names of costumes as JSON

### PR DESCRIPTION
**Pull request by wolfieboy09**
I made an extension that just returns the sprite's costume names.
JSON extension needed, because it returns an array

### CODE REVIEW

**Comment by wolfieboy09** [comment]
image

File: extensions/Wolfieboy09/spritecostumes.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// ID: costumer
+// Description: Block that returns the sprites costume names.
+// By: Wolfieboy09 <https://scratch.mit.edu/users/Wolfieboy09/>
+
+(function(Scratch) {
+    "use strict";
+	class Costumer {
+		getInfo() {
+			return {
+				id: 'costumer',
```
Comment (LilyMakesThings) [inline extensions/Wolfieboy09/spritecostumes.js:11]: The extension ID must contain your username

**Review by LilyMakesThings — CHANGES_REQUESTED** [review]
I'm not sure if there needs to be an entire extension for a single block

**Comment by GarboMuffin** [comment]
could this be a candidate for asset manager?

@LilyMakesThings

**Comment by GarboMuffin** [comment]
turns out this will be in Asset Manager *when* it is merged :)

**Pull request closed without merging**